source/deployment/customize-email.rst

```diff
@@ -0,0 +1,190 @@
+Email Templates
+====================
+
+Mattermost has a few email templates that are sent out when a specific event occurs.
+Most of the time these templates do not need to be modified.
+In case additional modifications are necessary, all available props in each email are listed below.  
+
+The email templates are located in the Mattermost server directory in the ``templates`` folder.
+The corresponding Strings for each prop can be found in the ``i18n`` folder. 
```

Comment:
Wondering if "Strings" needs to be capitalized here?

Reply:
That was my German coming through. Fixed, along with all the capitalized 'Text' occurences down below. Thanks!